contents/handbook/growth/marketing/product-announcements.md

```diff
@@ -4,72 +4,59 @@ sidebar: Handbook
+- Including them in the weekly changelog update and related emails.
+- Creating an in-app changelog notification.
 - Writing a Twitter and LinkedIn post.
-- Posting in the user Slack group.
-- Writing a tutorial created for week of release.
-- Creating an in-app banner for existing users.
-- Sharing links to the social media announcement internally via Slack, so colleagues can amplify them.
-- Sharing the announcement with other external parties, such as integration partners or PPC agencies.
 
 An example of a medium announcement includes the [launch of the NPS app](https://posthog.com/changelog/2023#nps-survey-app).
 
 ### Major announcements
-Major announcements involve changes which have a noticeable impact on the experience of most users, or require specific action from affected users. They may introduce new features, require product downtime, or include opt-in betas for upcoming work. 
-
-We may support major announcements by:
-
-- Including them in the changelog as soon as possible.
-- Including them in the monthly changelog email, and other regular emails.
-- Writing a Twitter thread and LinkedIn post.
-- Writing a Twitter thread to be posted by @james406.
-- Writing a dedicated blog post.
-- Sending targeted emails to all or affected users.
-- Posting in the PostHog user Slack group with a @here tag.
-- Writing a tutorial for day of release.
-- Creating a dedicated landing page.
-- Creating an in-app banner for existing users.
-- Creating on-site banner for all visitors.
-- Updating any third-party sites or listings, such as G2.com or StackShare.
-- Sharing links to the social media announcement internally via Slack, so colleagues can amplify them.
-- Sharing the announcement with other external parties, such as integration partners or PPC agencies.
+Major announcements involve changes which have a noticeable impact on the experience of most users, or require specific action from affected users. They may introduce new features, require product downtime, or include opt-in betas for upcoming work.
 
-We do not typically do public relations for anything other than company-level news. We have separate [processes and guides for managing press announcements](/handbook/growth/marketing/press). 
+We might do anything and everything for a major announcement.
 
 Examples of major announcements include [the surveys beta](/changelog/2023#user-surveys-beta) or [the analytics pricing change](/changelog/2023#reduced-pricing-for-product-analytics).
 
-## Announcing scheduled maintenance
+### New product announcements
+New product launches are major announcements. They have their own GitHub template: [Launch Plan](https://github.com/PostHog/meta/issues/new/choose). Marketing should always create a launch plan for new product announcements. 
+
+For new product announcements we generally apply the following best practices:
+
+- Ensure the product has at least one customer story created for it within 3 weeks of launch.
```

Comment:
Agree with having a customer story, but I'd probably put this at the bottom of the list priority-wise? We could even list this as a nice-to-have, whereas the rest of the list I'd consider table stakes.

Reply:
Oh, I didn't see these in order of priority - just all of them as nice to haves or guidelines. Think we can leave it on that basis for now.